js/index.js

```diff
@@ -25,6 +25,11 @@ document.addEventListener("DOMContentLoaded", function (event) {
         e.preventDefault();
         const searchVar = userSearch.value;
         console.log(searchVar);
+        
+        const deleteItems = document.querySelectorAll('.search-result__anime-content__item');
+        deleteItems.forEach(item => {
+            item.remove();
+        });
```

Comment:
Why do we need to perform the querySelector when we already have the `animeContent` element?

Reply:
Can't we just clear it's contents?
```javascript
animeContent.innerHTML = '';
```

Reply:
Oh, yeah. Thats much better!

Reply:
I fixed it in ae47074.